griptape/drivers/vector/redis_vector_store_driver.py

```diff
@@ -90,20 +91,47 @@ def query(
 
         results = self.client.ft(self.index).search(query_expression, query_params).docs
 
-        query_results = []
+        pipe = self.client.pipeline()
         for document in results:
-            metadata = getattr(document, "metadata", None)
-            namespace = document.id.split(":")[0] if ":" in document.id else None
-            vector_float_list = json.loads(document["vec_string"])
+            pipe.hget(document.id, "vector")
```

Comment:
Looks like this line executes an extra request to Redis for every result. Not sure we want that. Previous implementation didn't have that problem.